build/templates/examples.rst.mako

```diff
@@ -23,7 +23,9 @@
     if v.dev is None and v.pre is None:
         url = 'https://github.com/ni/nimi-python/releases/download/{0}/{1}_examples.zip'.format(global_version, module_name)
     else:
-        url = 'https://github.com/ni/nimi-python/blob/master/generated/examples/{0}_examples.zip'.format(module_name)
+        with open('./LATEST_RELEASE') as vf:
+            latest_release_version = vf.read().strip()
+        url = 'https://github.com/ni/nimi-python/releases/download/{0}/{1}_examples.zip'.format(latest_release_version, module_name)
```

Comment:
Why would this URL format be different than those of existing zip files in releases like https://github.com/ni/nimi-python/archive/1.1.5.zip?

Reply:
Because these are artifacts we are adding and not the automatic ones GitHub generates. I guess? These were the paths I found when I temporarily added a zip file to a release.